test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInitCommandTests.cs

```diff
@@ -158,7 +158,10 @@ public void InitCommand_Success_InvalidArguments_HelpMessage(string args)
                 waitForExit: true);
 
             // Assert
-            Util.VerifyResultSuccess(result, "usage: NuGet init <srcPackageSourcePath> <destPackageSourcePath> [options]");
+            // Why assert true ?
```

Comment:
Make sure to change the name of the test since it is now expecting failure instead of success.

`InitCommand_Failure_InvalidArguments_HelpMessage`